src/engine/enginepregain.cpp

```diff
@@ -107,10 +111,17 @@ void EnginePregain::process(CSAMPLE* pInOut, const int iBufferSize) {
     // Clamp gain to within [0, 10.0] to prevent insane gains. This can happen
     // (some corrupt files get really high replay gain values).
     // 10 allows a maximum replay Gain Boost * calculated replay gain of ~2
-    fGain = fGain * math_clamp(fReplayGainCorrection, 0.0f, 10.0f);
+    fGain *= math_clamp(fReplayGainCorrection, 0.0f, 10.0f);
 
     m_pTotalGain->set(fGain);
 
+    // We have to Vinyl emulation near stop, because otherwise we have clicks when
```

Comment:
This isn't quite accurate - the problem is not with clicking when reversing direction (the stretching algorithm actually handles this correctly).  Instead, say:

// As the speed approaches zero, hearing small bursts of sound at full volume is distracting and doesn't mimic the way that vinyl sounds when played slowly.  Instead, reduce gain to provide a soft rolloff.